widgets/GoogleSheets-batchsource.json

```diff
@@ -429,6 +429,22 @@
             "default": "100",
             "min": "1"
           }
+        },
+        {
+          "widget-type": "toggle",
+          "label": "Enable Cleansing Column Names",
+          "name": "columnNameCleansingEnabled",
+          "widget-attributes": {
+            "on": {
+              "value": "true",
+              "label": "Yes"
+            },
+            "off": {
+              "value": "false",
+              "label": "No"
+            },
+            "default": "true"
```

Comment:
Shouldn't default be `false`, when someone upgrades pipeline will this become `true` by default.

Users won't be happy updating old pipelines?

Reply:
Can you please add screenshot of plugin UI of one such old upgraded pipeline in PR description?